validator.py

```diff
@@ -50,13 +53,17 @@ def parse(filename):
         return
 
     for error in schema.iter_errors(pl):
+        pprint(error.absolute_path)
         post_error(error.message)
 
     foldernames = []
     displaynames = []
     repositories = []
 
-    os.mkdir("./" + bitness_from_input)
+    path = Path(bitness_from_input)
```

Comment:
Why is this change needed?

Reply:
I've just add a piece of code to clean up existing folder. It's useful when running several times the validator.
If you prefer, I can remove every changes in validator.py